project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.10.0"
 
-  private val workbenchLibsHash = "f31b537"
+  private val workbenchLibsHash = "8711e331-SNAP"
```

Comment:
TODO: update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/1561/ is merged